pkg/controller/storagecluster/reconcile.go

```diff
@@ -69,6 +70,16 @@ func (r *ReconcileStorageCluster) Reconcile(request reconcile.Request) (reconcil
 		// Error reading the object - requeue the request.
 		return reconcile.Result{}, err
 	}
+
+	ok, err := r.validateStorageCluster(instance)
```

Comment:
could we consider restructuring this a bit? Maybe call this function something like

```
isActive, err := isActiveStorageCluster(instance)
if err != nil {
// re-enqueue
// early return
}

if !isActive {
// make sure it's marked as phase=ignore
// update StorageCluster if necessary
// early return. 
}
```

I'd just like to see a clear path for handling errors vs a clear path for handling the situation where the StorageCluster isn't the current active StorageCluster.

Reply:
ack. Sounds good.